.github/workflows/e2e-cra-workflow.yml

```diff
@@ -9,6 +9,8 @@ on:
     - .github/actions/prepare/action.yml
     - .github/workflows/e2e-cra-workflow.yml
     - scripts/e2e-setup-ci.sh
+    - packages/yarnpkg-pnp/sources/hook.js
+    - packages/yarnpkg-pnp/sources/esm-loader/built-loader.js
```

Comment:
That will cause the E2E tests to run inside many PRs; the idea was to only run them inside the master branch, to avoid using too many GH Action resources. At the very least, should be moved to a separate PR to make a rollback easier.

Reply:
That was the point, to catch potential issues with changes to the PnP hook earlier.